dashboard/db/migrate/20170221204900_remove_activity_id_from_gallery_activities.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveActivityIdFromGalleryActivities < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :gallery_activities, :activity_id, :integer
+    remove_index :gallery_activities, name: :index_gallery_activities_on_user_id_and_activity_id
+  end
```

Comment:
Not sure I agree. Should the need arise to revert, I think we would prefer to accept the data loss than scramble to create a migration that adds the column and index back to the table.